### PR TITLE
Ds 64 display default image

### LINKED ISSUE #64: display default image for articles without image
**Is your feature request related to a problem? Please describe.**
Lack of image from the API should be handled with a default image.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. Display default images when API doesn't provide an article image
2. Fix alt tags for all images including decorative images

Closes #64

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Noice!

**Review by Gjhnsn — APPROVED** [review]
looks goooood

File: modern-news-app/src/components/TopNews.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { MainPageTitle } from "../style/SectionTitles.style";
 import Article from "./Article";
 import ExpandedArticle from "./ExpandedArticle";
 import { ArticlesStyled } from "./Main";
+import noimageavailable from "../assets/noimageavailable.jpg";
```
Comment (Lap343) [inline modern-news-app/src/components/TopNews.tsx:7]: is this import necessary if it is not used in the file?

**Review by Lap343 — APPROVED** [review]
Other than the random import I would say this code is good to go.

**Review by Wolftownn — APPROVED** [review]
Everything looks good other than the unused imported jpg file.

**Review by BlackCubes — APPROVED** [review]
Looks gewd!

**Review by krferrell — APPROVED** [review]
Nice touch!

**Pull request merged**